src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -1715,6 +1731,105 @@ def aks_agentpool_delete_machines(cmd,   # pylint: disable=unused-argument
     )
 
 
+def aks_agentpool_manual_scale_add(cmd,
+                                   client,
+                                   resource_group_name,
+                                   cluster_name,
+                                   nodepool_name,
+                                   vm_sizes,
+                                   node_count,
+                                   no_wait=False):
+    instance = client.get(resource_group_name, cluster_name, nodepool_name)
+    if instance.type_properties_type != CONST_VIRTUAL_MACHINES:
+        raise CLIError("Cannot add manual to a non-virtualmachines node pool.")
+    ManualScaleProfile = cmd.get_models(
+        "ManualScaleProfile",
+        resource_type=CUSTOM_MGMT_AKS_PREVIEW,
+        operation_group="managed_clusters",
+    )
+    new_manual_scale_profile = ManualScaleProfile(sizes=vm_sizes.split(","), count=int(node_count))
```

Comment:
`vm_sizes.split(",")` seems not safe enough, user may provide value with extra space, if server cannot handle this well, may results in unexpected behavior

Reply:
if user provides spaces in `--node-vm-size`, server will also return error. I think it's expected.